server/db/migration/public/migrations/sqls/20210329113834-add-table-user-invitation-up.sql

```diff
@@ -0,0 +1,22 @@
+/* Replace with your SQL commands */
+
+CREATE EXTENSION IF NOT EXISTS "postgis";
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE
+  "user_invitation"
+(
+  uuid           uuid        NOT NULL DEFAULT uuid_generate_v4(),
+  user_uuid      uuid        NOT NULL,
+  survey_uuid    uuid        NULL,
+  invited_by     uuid        NOT NULL,
+  invited_date   TIMESTAMP   NOT NULL DEFAULT (now() AT TIME ZONE 'UTC'),
+  removed_date   TIMESTAMP   NULL,
+
+
+
+  PRIMARY KEY ("uuid"),
+  CONSTRAINT user_invitation_user_fk FOREIGN KEY (user_uuid) REFERENCES "user" ("uuid") ON DELETE CASCADE,
+  CONSTRAINT user_invitation_survey_fk FOREIGN KEY (survey_uuid) REFERENCES "survey" ("uuid") ON DELETE CASCADE,
+  CONSTRAINT user_invitation_invited_by_fk FOREIGN KEY (invited_by) REFERENCES "user" ("uuid") ON DELETE CASCADE
+);
```

Comment:
you can add this script to the 20180709092931-create-user-table-up.
we're still in dev mode and it's safe to update existing ones

Reply:
I need to do after the survey creation

Reply:
right!